sw/device/tests/sim_dv/keymgr_key_derivation_test.c

```diff
@@ -174,8 +58,6 @@ bool test_main(void) {
   } else if (info == kDifRstmgrResetInfoSw) {
     LOG_INFO("Powered up for the second time, actuate keymgr");
 
-    init_kmac_for_keymgr();
```

Comment:
According to @eunchan you'd better not use the default entropy_mode in kmac. I plan to select either edn_mode or sw_mode in this function, prior to start any keymgr operation.

If you have a change, can you help to add that? can be in a separate PR.

Reply:
My next PR has a `kmac_testutils.c` with a configuration routine; in the meantime, I added back the `init_kmac_for_keymgr()` here and can just replace it on the next PR.

Reply:
sounds good. Thanks Jade